package.json

```
@@ -29,27 +29,35 @@
   "qna": "https://r2c.dev/slack",
   "main": "./out/src/extension.js",
   "contributes": {
-    "viewsContainers": {
-      "activitybar": [
-        {
-          "id": "semgrep-explorer",
-          "title": "Semgrep Explorer",
-          "icon": "semgrep-icon.png"
+    "icons": {
+      "semgrep-icon": {
+        "description": "Semgrep Logo",
+        "default": {
+          "fontPath": "./semgrep-icon.woff",
+          "fontCharacter": "A"
         }
-      ]
+      }
     },
     "commands": [
       {
         "command": "semgrep.login",
         "title": "Semgrep: Login"
```

```suggestion
        "title": "Semgrep: Sign in"
```

```suggestion
        "title": "Semgrep: Sign out"
```

same words as on semgrep.dev